addons/test_mail/tests/test_mail_alias.py

```diff
@@ -807,6 +807,17 @@ def test_copy_content(self):
         record_copy2 = record_copy.copy()
         self.assertEqual(record_copy2.alias_bounced_content, new_content)
 
+    @users('employee')
+    def test_copy_optional_alias_model(self):
```

Comment:
We could probably add a small docstring to know why it is there, like
```suggestion
    def test_copy_optional_alias_model(self):
        """ Do not propagate alias_id to duplicate record as it would lead to
        overwriting alias_name of old record. """
```

Reply:
Done! 
Thanks for taking the time to look into this.